changelog.md

```diff
@@ -1,3 +1,78 @@
+# CITYENGINE SDK 3.2.10646 CHANGELOG
+
+This section lists changes compared to CityEngine SDK 3.2.10352.
+
+## General Info
+* CityEngine SDK 3.2.10646 is used in CityEngine 2024.1.10646.
+
+## PRT API
+* `prt::Attributable::toXML()`: Fixed a bug where some float numbers were converted to wrong string values in rare cases. Further, the maximum number of digits in the string representation was reduced by 1.  
+
+## PRTX API
+* `prtx::EncodePreparator`: Fixed a bug in merge-by-material setups where vertex normals for faces with holes were wrong when meshes with vertex nomals were merged with meshes without vertex normals. 
+
+## CGA
+* New features:
+  * inline keyword: Allows to immediately generate a rule and continue working on the resulting shape(s). This means any rule can be used like a builtin shape operation.
+  * union, subtract, intersect keywords: added Boolean 3D operations.
+  * extrude operation: Added support to extrude edges.
+* New operations:
+  * tag operation
+* Changes to existing features:
+  * i operation: The new parameter cleanupMode allows for disabling the numerical geometry cleanup for inserted asset. This improves the support for assets with tiny details.
+  * cleanupGeometry operation: Removing coplanar edges now preserves the first edge.
+  * Floating point precision:
+      * All internal geometry processing was switched form float to double precision. The numerical behaviour is more accurate now. It is possible that some rules change behaviour because e.g a float value comparison gives a different result now.
```

Comment:
e.g.

Reply:
didn't change it, I'd like to keep this as a 1:1 copy of the changelog in the CGA ref